source/config.js

```diff
@@ -212,3 +214,22 @@ Object.defineProperty(Error.prototype, 'toJSON', {
   },
   configurable: true
 });
+
+try {
+  module.exports.gitVersion = /.*?(\d+[.]\d+[.]\d+).*/.exec(child_process.execSync('git --version').toString())[1];
```

Comment:
I do favor async calls above sync calls generally but for this case it was much easier to do it this way and seems like unnecessary complication for little benefits.  I don't think start up time was hindered very much.